libbeat/_meta/fields.ecs.yml

```diff
@@ -1126,12 +1129,12 @@
     - name: response.bytes
       level: extended
       type: long
-      format: bytes
```

Comment:
Are we missing this in ECS? If yes, maybe just put it back in and fix ECS?

Reply:
Ah, thanks for catching that. Will re-add. This format is already in ECS master (1.1.0-dev).

We didn't intend to backport the `format:bytes` to 1.0 when we added it in https://github.com/elastic/ecs/pull/385, because we expected ECS 1.1 would be ready by then. Since that didn't pan out, there is this single remaining difference between ECS 1.0.1 and Beats' copy of the file.

I only realized that after cutting 1.0.1 earlier today 😢